models.R

```diff
@@ -92,6 +97,7 @@ if (config$targets[index] == "ILI ED visits" && config$regions_to_fit[index] ==
     trend_formula = ~
       # Hierarchical intercepts capture variation in average count
       s(trend, bs = "re") +
+        s(trend, season, bs = "re") +
```

Comment:
does this introduce an intercept for trend on its own (i.e is it a duplicate of line 99). I find the syntax very confusing personally. The way to check is to look at the output I think